app/services/DynamoChannelTests.scala

```diff
@@ -56,6 +56,7 @@ class DynamoChannelTests(stage: String) extends StrictLogging {
         BatchWriteItemRequest
           .builder
           .requestItems(Map(tableName -> writeRequests.asJava).asJava)
+          .returnConsumedCapacity(ReturnConsumedCapacity.TOTAL)
```

Comment:
this is just to get it to show in our logs how many units it used during batch writes